knockout_modeler/ko.py

```diff
@@ -44,38 +47,38 @@ def get_object_data(obj, fields, safe):
     temp_dict = dict()
     for field in fields:
 
-            try:
-                attribute = getattr(obj, str(field))
-                if isinstance(attribute, list) and \
-                        all([isinstance(item, (dict, models.Model, fake_models.FakeModel)) for item in attribute]):
-                    temp_dict[field] = []
-                    for item in attribute:
-                        temp_dict[field].append(get_object_data(item, get_fields(item), safe)) # Recur
-                elif isinstance(attribute, (dict, models.Model, fake_models.FakeModel)):
-                    attribute_fields = get_fields(attribute)
-                    object_data = get_object_data(attribute, attribute_fields, safe) # Recur
-                    temp_dict[field] = object_data
-                else:
-                    if not safe:
-                        if isinstance(attribute, basestring):
-                            attribute = cgi.escape(attribute)
-                    temp_dict[field] = attribute
-
-            except Exception, e:
-                logger.info("Unable to get attribute.")
-                logger.error(e)
-                continue
+        try:
+            attribute = getattr(obj, str(field))
+            if isinstance(attribute, list) and all([isinstance(item, models.Model) for item in attribute]):
```

Comment:
The block was de-indented, and ignoring white-space will only show the following changes (line 52 and 56 below, on the right-side) that were cause of the bugs:

- `isinstance(item, (dict, models.Model, fake_models.FakeModel))` is replaced by `isinstance(item, models.Model)` as calling `get_fields` on a `dict` results in `[]`. `FakeModel` instance check is redundant as [it itself inherits from `models.Model`](https://github.com/erm0l0v/django-fake-model/blob/master/django_fake_model/models.py#L10).